diff --git a/base/BUILD.gn b/base/BUILD.gn
index d761617279da9d3769968d06500d662d3f5ed72f..01bed3a192be7695d7c031d38f232a5184bd9923 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5233,6 +5233,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5925,6 +5926,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
