diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
index 92752e4a29fa7a2fbc1941dbe2519144b642e4bf..19002a8d19b19f447a0b24b3ef20d433c8f6759f 100644
--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -1787,6 +1787,7 @@ static_library("ui") {
     # split out yet.
     "//chrome/browser/webauthn:impl",
   ]
+  import("//brave/browser/sources.gni") public_deps += [ "//brave/browser/ui" ] deps += brave_chrome_browser_ui_allow_circular_includes_from + brave_chrome_browser_ui_deps allow_circular_includes_from += brave_chrome_browser_ui_allow_circular_includes_from
 
   if (!is_android) {
     deps += [
@@ -4260,6 +4261,7 @@ group("ui_public_dependencies") {
     "//components/enterprise/buildflags",
     "//components/paint_preview/buildflags",
     "//components/safe_browsing:buildflags",
+    "//components/saved_tab_groups/public",
     "//components/segmentation_platform/public",
     "//components/sync",
     "//components/sync_user_events",
@@ -4272,6 +4274,7 @@ group("ui_public_dependencies") {
   if (enable_extensions_core) {
     public_deps += [ "//extensions/browser" ]
   }
+  public_deps += [ "//brave/browser/ui:ui_public_dependencies" ]
   if (!is_android) {
     public_deps += [
       "//build:branding_buildflags",
