diff --git a/chrome/browser/ui/autofill/BUILD.gn b/chrome/browser/ui/autofill/BUILD.gn
index d37d11ddeeead4185e1387d1602b7a0de54c1a19..0ca97b9ad42b09ddbadb2f1bca1af4c1456b1acb 100644
--- a/chrome/browser/ui/autofill/BUILD.gn
+++ b/chrome/browser/ui/autofill/BUILD.gn
@@ -254,6 +254,7 @@ source_set("impl") {
     "//components/variations/service",
     "//components/vector_icons",
   ]
+  import("//brave/chromium_src/chrome/browser/ui/autofill/sources.gni")  deps += brave_chromium_src_chrome_browser_ui_autofill_deps
 
   if (safe_browsing_mode != 0) {
     deps += [ "//components/safe_browsing/content/browser/password_protection" ]
@@ -280,6 +281,7 @@ source_set("impl") {
       "//components/messages/android:feature_flags",
       "//components/password_manager/core/browser/features:password_features",
     ]
+    import("//brave/browser/ui/autofill/sources.gni") deps += brave_chrome_browser_ui_autofill_deps
   } else {
     sources += [
       "address_bubbles_controller.cc",
