diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 899a9570ceeca34aa2dcb5acb2c5146c1a9df2fb..2a569cf964bb5762ebb9b6de24b183d2aec7cb52 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1409,3 +1409,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUiManager.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
