diff --git a/chrome/android/expectations/expectations.gni b/chrome/android/expectations/expectations.gni
index 7f7fdb9d103162a321914a5c495326cdabdd3b49..b01c6a626ca0a810439bb7452f12589c9b98d26d 100644
--- a/chrome/android/expectations/expectations.gni
+++ b/chrome/android/expectations/expectations.gni
@@ -7,7 +7,7 @@ import("//build/config/coverage/coverage.gni")
 
 # Chosen to match android-binary-size trybot.
 enable_libs_and_assets_verification =
-    !enable_chrome_android_internal && !is_component_build &&
+    false && !enable_chrome_android_internal && !is_component_build &&
     !use_jacoco_coverage && is_official_build &&
     ((target_cpu == "arm64" || target_cpu == "x64") &&
      android_channel == "stable")
@@ -17,6 +17,6 @@ enable_libs_and_assets_verification =
 # android_channel: Manifest entries vary based on channel.
 # target_cpu: android:versionCode varies.
 enable_manifest_verification =
-    !is_java_debug && !enable_chrome_android_internal && !use_jacoco_coverage &&
+    false && !is_java_debug && !enable_chrome_android_internal && !use_jacoco_coverage &&
     ((target_cpu == "arm64" || target_cpu == "x64") &&
      android_channel == "stable")
