diff --git a/DEPS b/DEPS
index e26f60ed95936b4fa31fd11ff00731677db3169e..6785c82f8a59cbffb1569a2a1a0921bbfc0e51ec 100644
--- a/DEPS
+++ b/DEPS
@@ -4537,7 +4537,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
