diff --git a/front_end/models/ai_assistance/BUILD.gn b/front_end/models/ai_assistance/BUILD.gn
index c49c747cb6120045dca1f51e2e829ba43e308d90..360ce67b57785bb0eac6623ec69772405c2db113 100644
--- a/front_end/models/ai_assistance/BUILD.gn
+++ b/front_end/models/ai_assistance/BUILD.gn
@@ -112,6 +112,7 @@ devtools_module("ai_assistance") {
 }
 
 devtools_entrypoint("bundle") {
+  deps = [ "skills:generate_skills" ]
   split_compilation = true
   entrypoint = "ai_assistance.ts"
 
